packages/storybook-vue/src/stories/Textarea/Textarea.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Textarea } from "@utrecht/component-library-vue";
+import { argTypes } from "./argTypes.ts";
+
+<Meta id="vue-textarea" title="Vue.js Component/Textarea" component={Textarea} argTypes={argTypes} />
+
+# Textarea
+
+export const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { Textarea },
+  setup() {
+    return { args };
+  },
+  template: `
+  <FormField> 
+    <Textarea v-bind="args" @update:modelValue="action" :value="args.modelValue" />
+  <br/>
+  <br/>
+  <Paragraph v-if="args.modelValue"><strong> Textarea Binding:</strong> {{args.modelValue}}</Paragraph>
+  </FormField>
+  `,
+  methods: { action: action("modelValue") },
+});
```

Comment:
I suggest we keep the default story of every component quite simple, and include a separate page in Storybook that shows form bindings for form components.